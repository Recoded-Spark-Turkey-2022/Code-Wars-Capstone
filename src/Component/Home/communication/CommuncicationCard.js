import Card from "./card";
import Chat from "./Imges/chat.svg" 
import PhoneCall from "./Imges/phonecall.svg" 
import Video from "./Imges/video.svg" 



function CommuncicationCard(){
return (
 
    <>
    <p className="text-4xl absolute translate-y-10 translate-x-20 ">
     WE CAN COMMUNICATE THROUGH
    </p>
        <div className="flex gap-8 mx-20 translate-y-1/3 mt-20 ">
    <Card  imageUrl={PhoneCall} header="Voice Call" text="Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!" />
    <Card  imageUrl={Chat} header="Chat" text="Need to talk to someone? Come have a chat with your therapist!" />
    <Card imageUrl={Video} header="Video Call" text="For better experience therapists recommend video calls, but always remember that its a choice!" />
    </div>
    </>
  
   
   
)
}
export default CommuncicationCard;