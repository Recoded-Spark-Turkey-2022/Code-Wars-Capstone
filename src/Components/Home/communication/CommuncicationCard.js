import Card from "./card";
import Chat from "./Imges/chat.svg" 
import PhoneCall from "./Imges/phonecall.svg" 
import Video from "./Imges/video.svg" 



function CommuncicationCard(){
return (
 
    < >
    <p style={{fontFamily: 'Poppins, sans-serif'}} className="text-4xl my-20 ml-40  ">
     WE CAN COMMUNICATE THROUGH
    </p>
        <div className="flex gap-8  mb-20 mx-40    ">
    <Card  imageUrl={PhoneCall} header="Voice Call" text="Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!" />
    <Card  imageUrl={Chat} header="Chat" text="Need to talk to someone? Come have a chat with your therapist!" />
    <Card imageUrl={Video} header="Video Call" text="For better experience therapists recommend video calls, but always remember that its a choice!" />
    </div>
    </>
  
   
   
)
}
export default CommuncicationCard;