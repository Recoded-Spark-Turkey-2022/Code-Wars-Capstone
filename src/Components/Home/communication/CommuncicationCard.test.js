
import React from "react";
import renderer from "react-test-renderer";
import CommuncicationCard from "./CommuncicationCard";
import { MemoryRouter } from "react-router-dom";

it(" CommuncicationCard renders correctly", () => {
    const test = renderer
        .create(
            <MemoryRouter>
                <CommuncicationCard/>
            </MemoryRouter>
        )
        .toJSON();
    expect(test).toMatchSnapshot();
});