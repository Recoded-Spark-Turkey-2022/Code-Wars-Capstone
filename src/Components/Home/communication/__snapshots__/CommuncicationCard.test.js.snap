// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` CommuncicationCard renders correctly 1`] = `
Array [
  <p
    className="text-4xl my-20 ml-40  "
    style={
      Object {
        "fontFamily": "Poppins, sans-serif",
      }
    }
  >
    WE CAN COMMUNICATE THROUGH
  </p>,
  <div
    className="flex gap-8  mb-20 mx-40    "
  >
    <div
      className="bg-white rounded-lg drop-shadow-xl p-6 w-10/12"
      style={
        Object {
          "fontFamily": "Poppins, sans-serif",
        }
      }
    >
      <img
        alt="cardsphoto"
        className="h-20 w-15 rounded-full mx-auto mb-4"
        src="phonecall.svg"
      />
      <h2
        className="text-2xl font-bold text-center mb-2"
      >
        Voice Call
      </h2>
      <p
        className="text-gray-700 text-center"
      >
        Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!
      </p>
    </div>
    <div
      className="bg-white rounded-lg drop-shadow-xl p-6 w-10/12"
      style={
        Object {
          "fontFamily": "Poppins, sans-serif",
        }
      }
    >
      <img
        alt="cardsphoto"
        className="h-20 w-15 rounded-full mx-auto mb-4"
        src="chat.svg"
      />
      <h2
        className="text-2xl font-bold text-center mb-2"
      >
        Chat
      </h2>
      <p
        className="text-gray-700 text-center"
      >
        Need to talk to someone? Come have a chat with your therapist!
      </p>
    </div>
    <div
      className="bg-white rounded-lg drop-shadow-xl p-6 w-10/12"
      style={
        Object {
          "fontFamily": "Poppins, sans-serif",
        }
      }
    >
      <img
        alt="cardsphoto"
        className="h-20 w-15 rounded-full mx-auto mb-4"
        src="video.svg"
      />
      <h2
        className="text-2xl font-bold text-center mb-2"
      >
        Video Call
      </h2>
      <p
        className="text-gray-700 text-center"
      >
        For better experience therapists recommend video calls, but always remember that its a choice!
      </p>
    </div>
  </div>,
]
`;
